lib/elixir/pages/Guards.md

```diff
@@ -191,11 +200,11 @@ def my_function(number) when is_integer(number) and rem(number, 2) == 0 do
 end
 ```
 
-This would be repetitive to write every time we need this check, so, as mentioned at the beginning of this section, we can abstract this away using a macro. Remember that defining a function that performs this check wouldn't work because we can't use custom functions in guards. Use `defguard` and `defguardp` to create guard macros. Here's an example:
+It would be repetitive to write every time we need this check, so, as mentioned at the beginning of this section, we can abstract this away using a macro. Remember that defining a function that performs this check wouldn't work because we can't use custom functions in guards. Instead you can use `defguard/1` and `defguardp/1` to create guard-safe macros. Here's an example how:
```

Comment:
"guard-safe" is introduced here but it is not explained what it is anywhere else. I would revert the `-safe` addition.

Reply:
removed the use of guad-safe throughout the page.